python/ray/tune/test/trial_runner_test.py

```diff
@@ -142,17 +142,16 @@ def _train(self):
         register_trainable("B", B)
 
         def f(cpus, gpus, queue_trials):
-            return run_experiments(
-                {
-                    "foo": {
-                        "run": "B",
-                        "config": {
-                            "cpu": cpus,
-                            "gpu": gpus,
-                        },
-                    }
-                },
-                queue_trials=queue_trials)[0]
+            return run_experiments({
+                "foo": {
+                    "run": "B",
+                    "config": {
+                        "cpu": cpus,
+                        "gpu": gpus,
+                    },
+                }
+            },
+                                   queue_trials=queue_trials)[0]
```

Comment:
I think this linting is wrong? be sure to install the right yapf version (0.23)

Reply:
ah i see, i was on 0.25